### ISSUE TITLE
consider adding nmds

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash_plugin_betterplot#44]
https://scikit-learn.org/stable/modules/generated/sklearn.manifold.MDS.html - set `metric=False`
